src/lib/stitching/mergeSchemas.ts

```diff
@@ -0,0 +1,59 @@
+import { mergeSchemas as _mergeSchemas } from "graphql-tools"
+import { executableConvectionSchema } from "lib/stitching/convection/schema"
+import { consignmentStitchingEnvironment } from "lib/stitching/convection/stitching"
+import { executableGravitySchema } from "lib/stitching/gravity/schema"
+import _ from "lodash"
+
+import localSchema from "../../schema"
+
+const scalarsToKeepInMetaphysics = [
+  "String",
+  "ID",
+  "Int",
+  "Boolean",
+  "PageInfo",
+]
+
+export const mergeSchemas = async () => {
+  const convectionSchema = await executableConvectionSchema()
+  const convectionStitching = consignmentStitchingEnvironment(
+    localSchema,
+    convectionSchema
+  )
+
+  const gravitySchema = await executableGravitySchema()
+
+  // Add gravity schema first to prefer local MP schema.
+  // Add schemas after localSchema to prefer those over MP.
+  const mergedSchema = _mergeSchemas({
+    schemas: [
+      gravitySchema,
+      localSchema,
+      convectionSchema,
+      convectionStitching.extensionSchema,
+    ],
+    onTypeConflict: (leftType, rightType) => {
```

Comment:
Curious about this- `gravitySchema` has `Artwork` and other conflicting types, so based on this handler now, are we getting errors in development now?

ie- I recall seeing `Type Collision Artwork` and the like in the existing version. Since we want to keep those from Metaphysics, should we add them to `scalarsToKeepInMetaphysics` (and rename it to drop 'scalar')?

Reply:
I took out merging of gravity's graphql for this as it's not being used, will figure that one out later but we will probably need to either do a full switcheroo with MP's `Artist`, or rename Grav's GraphQL for the time being. 

WRT scalars, those are core primitives that are definitely the same, merging and overwriting we should be handling on a case by case basis

Reply:
Wait, but we are still merging in gravity GraphQL? (or am i misunderstanding the term 'merging' and 'stitching', etc. again 😆 )

Also, there is a mutation we added in Gravity's GraphQL that we wanted to use transparently thru Metaphysics (`recordArtworkView`).

Reply:
Ah, I'm surprised that removing it didn't break a test then, sure I can get it merged back in with #1032

Reply:
ah, that makes sense then yeah! Cool, I'll re-create a version of that test that uses stitching 👍